flake.nix

```diff
@@ -400,6 +408,8 @@
             };
 
             docs = project.buildSearchablePursDocs;
+
+            ogmios-fixtures = project.buildOgmiosFixtures { };
```

Comment:
Do we need this here? `buildOgmiosFixtures` is being created in several different places

Reply:
Mostly left over from my development of this feature, it was so I could look at the output of my fixture generating nix on its own. Happy to remove